### PR TITLE
Edited IDs in logos files

### PR DESCRIPTION
**Pull request by adamsulimadolina**
I had problem with rendering some logos - sometimes colors just dissapeared. I asked on StackOverflow and according to [this](https://stackoverflow.com/a/64849228/11213800) asnwer, I've changed IDs in logos files (e.x for Boston Celtics from id:"A" to id:"A_BOS"). It started to work properly, so I hope it was correct solution.

### CODE REVIEW

**Comment by ChrisKatsaras** [comment]
Hi @adamsulimadolina . Sorry for the late reply! I didn't get a notification for this. I don't see any issue with making this change but I just have 1 question for you, did you have to manually change all of these files? I have 2 other repos for MLB and NFL logos that I would need to update as well based on this change and I would like to know if there's an easy way to do it

**Comment by ChrisKatsaras** [comment]
Just out of curiosity, could we not just have updated the IDs as suggested to fix the issue instead of updating the entire structure of the file as well? @adamsulimadolina

**Comment by ChrisKatsaras** [comment]
After doing some searching, it appears that my other two repos do not have any references to IDs in the SVG files, so I think I'm fine there. So unless there's a good reason to update the entire structure of the file, I suggest we just update the IDs in these SVG files, and then I'll merge your PR / republish so everyone can benefit from your change

**Comment by adamsulimadolina** [comment]
Yeah, I edited every file manually. If you want you can update structure file, I just get this idea from stack, so I posted it here :)

**Comment by ChrisKatsaras** [comment]
Fair enough! I think I'll make a PR where we just update the IDs but keep the file structure the same 😄  I'll try and do it tonight if I have time! Thanks for the help on this @adamsulimadolina and I'm glad that you're using my NPM package!

**Comment by ChrisKatsaras** [comment]
@adamsulimadolina I see what you did here now. So you were on the right track updating the IDs but I now understand why the format is so different. From what I can tell, you ran my prod script to generate the files and then replaced the src ones (and updated the IDS). This isn't what we want as we want to have a "dev" copy (located within src folder) and a prod copy (generated using my prod script in package.json). I run the prod script before pushing to the npm registry which not only creates the prod folder but removes the unnecessary src folder (since people using this package don't need it)

However, I see that I made a mistake the last time I published this package though and forgot to remove the SRC folder (doubling the number of files in the package). I'll make a PR tonight to fix this (along with updating IDs) 😄 

TLDR: We want to maintain a dev set of files and only generate the more efficient prod files when we want to publish to NPM. Also, the prod version files are far less readable which is why I maintain a dev set. 
PR will be up soon to fix this
